codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/endpoint/generators/EndpointParamsInterceptorGenerator.kt

```diff
@@ -83,10 +83,10 @@ class EndpointParamsInterceptorGenerator(
             let input = context.input()?;
             let _input = input
                 .downcast_ref::<${operationInput.name}>()
-                .ok_or_else(|| #{InterceptorError}::invalid_input_access())?;
+                .ok_or_else(|| "failed to downcast to ${operationInput.name}")?;
             let params_builder = cfg
                 .get::<#{ParamsBuilder}>()
-                .ok_or(#{InterceptorError}::read_before_execution("missing endpoint params builder"))?
+                .ok_or_else(|| "missing endpoint params builder".to_owned())?
```

Comment:
Is the `to_owned()` needed? Looks like it wasn't on the one above.

Reply:
Good catch. Fixed in 51026e9.